### PR TITLE
PAS: Removed dependency on policy.json in the Proxy Attestation Server CLI

### PR DESCRIPTION
**Pull request by geky**
This was a lingering limitation from when the Proxy Attestation Server required more info from the policy file.

Now, in `vc-pas`, the primary argument is what address to bind to, and no policy is needed in the Proxy Attestation Server.

It's almost possible to let users omit this argument, and default to the standard Proxy Attestation Port for all addresses, except, well, we don't have a standard port for the Proxy Attestation Server. Choosing two arbitrary ports as standard ports for Veracruz's Proxy Attestation Server and Server Server would be nice for quality-of-life things like this, but I've omitted it for now.

### CODE REVIEW

File: proxy-attestation-server/src/server.rs
Comment (dreemkiller) [inline proxy-attestation-server/src/server.rs:199]: Is there something that requires `U`, `P1`, and `P1` instead of using the types directly? Two levels of indirection to find out what types the parameters are seems a bit much if it can be avoided.

**Review by dominic-mulligan-arm — DISMISSED** [review]

**Comment by dominic-mulligan-arm** [comment]
LGTM, though I'll wait for @dreemkiller's approval before this can be merged.

File: proxy-attestation-server/src/server.rs
Comment (geky) [reply proxy-attestation-server/src/server.rs:199]: Strictly no, these could be changed to a bunch of `&str`s. But that would move more work into the user to convert down to `&str` when HttpServer::bind/File::open are going to take U:ToSocketAddrs/P:AsRef<Path> anyways. It would also be a little bit less type safe, using `&str` for everything instead of specific traits. 

As for the indirection, that's sort of on Rust's syntax. The traits could be inlined in this case (not always), but the syntax of traits require at least one indirection without changing the behavior of the API:

``` rust
pub fn server<
    U: ToSocketAddrs,
    P1: AsRef<path::Path>,
    P2: AsRef<path::Path>
>(
    url: U,
    ca_cert_path: P1,
    ca_key_path: P2,
    debug: bool
) -> Result<Server, String> {
    blablabla
```

**Comment by alexandref75** [comment]
Can't comment on the code (not a rust programmer) but the functionality LGTM.

File: proxy-attestation-server/src/server.rs
Comment (alexandref75) [reply proxy-attestation-server/src/server.rs:199]: The functionality LGTM. Cannot check the code.

**Comment by dominic-mulligan-arm** [comment]
@dreemkiller are you happy with @geky's explanation above?

**Pull request merged**
